aggregators/metamask.ts

```diff
@@ -112,6 +113,17 @@ const adapter: Adapter = {
     [CHAIN.LINEA]: { start: '2023-10-03', },
   },
   methodology,
+  breakdownMethodology: {
+    Fees: {
+      [METRIC.TRADING_FEES]: "All fees paid by users for trading, swapping, bridging in Metamask wallet.",
```

Comment:
I am in two minds about how big these should be, if it is showing up on the reports, shorter the better? 
for complicated stuff, we can add another field that gives more info when you hover over it?